webapp/src/components/sidebar_right/github_items.tsx

```diff
@@ -5,10 +5,10 @@ import * as React from 'react';
 
 import * as CSS from 'csstype';
 
-import {Badge, Tooltip, OverlayTrigger} from 'react-bootstrap';
+import {Badge, OverlayTrigger, Tooltip} from 'react-bootstrap';
 import {Theme} from 'mattermost-redux/types/preferences';
-import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
-import {GitPullRequestIcon, IssueOpenedIcon, IconProps} from '@primer/octicons-react';
+import {changeOpacity, makeStyleFromTheme} from 'mattermost-redux/utils/theme_utils';
+import {GitPullRequestIcon, IconProps, IssueOpenedIcon} from '@primer/octicons-react';
```

Comment:
None of these changes seem to be necessary for this PR. Can you please double check your changes when submitting the PRs to ensure all changes are necessary for the ticket?

Reply:
They are probably changed by ESLint, since imports are same. Just sequence is changed. do you want me to revert this part?

Reply:
They are probably changed by ESLint, since imports are same. Just sequence is changed. do you want me to revert this part?

Reply:
fixed with https://github.com/mattermost/mattermost-plugin-github/pull/462#:~:text=commit%20by%20sanjay-,443aa19,-Merge%20state